bundlewrap/items/users.py

```diff
@@ -134,18 +134,33 @@ def cdict(self):
 
     def fix(self, status):
         if status.must_be_deleted:
-            self.run("userdel {}".format(self.name), may_fail=True)
+            if self.node.os == 'freebsd':
+                command = "pw userdel {}"
+            else:
+                command = "userdel {}"
+            self.run(command.format(self.name), may_fail=True)
         else:
-            command = "useradd " if status.must_be_created else "usermod "
+            if self.node.os == 'freebsd':
+                command = "pw useradd " if status.must_be_created else "pw usermod "
+                command += self.name + " "
+            else:
+                command = "useradd " if status.must_be_created else "usermod "
             for attr, option in sorted(_ATTRIBUTE_OPTIONS.items()):
                 if (attr in status.keys_to_fix or status.must_be_created) and \
                         self.attributes[attr] is not None:
                     if attr == 'groups':
                         value = ",".join(self.attributes[attr])
+                    elif attr == 'password_hash' and self.node.os == 'freebsd':
+                        # On FreeBSD, pw useradd/usermod -p sets the password expiry time.
+                        # Using -H <n> we pass the password hash using file descriptor <n> instead.
+                        option = '-H'
+                        value = '0'
+                        command = "printf '%s' '{}' | ".format(self.attributes[attr]) + command
                     else:
                         value = str(self.attributes[attr])
                     command += "{} {} ".format(option, quote(value))
-            command += self.name
+            if self.node.os != 'freebsd':
+                command += self.name
```

Comment:
Let's just do it that way for Linux as well, `usermod trehn -c test` works just fine.